index.js

```diff
@@ -18,6 +18,8 @@ app.use(require("errorhandler")(errorhandlerOptions));
 // bibrefs
 app.get('/bibrefs', function (req, res, next) {
     var refs = req.query["refs"];
+    res.setHeader("Expires", new Date(Date.now() + 86400000).toUTCString());
```

Comment:
Yeah, It's rare that we get manual update requests nowadays, so I guess the use-case where someone just fixed the db and wants to see the update right away in their spec is rare enough.